tests/test_dispatcher_priority.py

```
@@ -0,0 +1,238 @@
+"""Tests for the dispatcher priority ordering."""
+
+from __future__ import annotations
+
+from unittest.mock import MagicMock, AsyncMock
```

Flake8 (7.3.0)

[error] 'unittest.mock.AsyncMock' imported but unused

(F401)